doc/pic/hld_fpmsyncd.md

```diff
@@ -248,6 +262,32 @@ Therefore, even after this enhancement, table entries will be created for `ROUTE
 ##### Figure: Example of ASIC_DB entry
 ![fig3](images_fpmsyncd/fig3.svg)
 
+#### Orchestration Agent Changes to handle NEXTHOP_GROUP
+
```

Comment:
Please capture the case NH as Interface?

Reply:
"NH as Interface" will be same as any other NHG.  The application ( zebra) will create a NHID for the NHG with NH as interface and the route will use this NHID as reference to the NH